istio/istio.go

```diff
@@ -383,7 +392,49 @@ func (iClient *IstioClient) executeBookInfoInstall(ctx context.Context, arReq *m
 	return nil
 }
 
-// ApplyRule is a method invoked to apply a particular operation on the mesh in a namespace
+
+func (iClient *IstioClient)executeHipsterShopInstall(ctx context.Context, arReq *meshes.ApplyRuleRequest) error {
+
+	resp, err := http.Get(hipsterShopKubernetesManifestsURL)
+	if err != nil {
+		err = errors.Wrapf(err, "error parsing response body")
+		logrus.Error(err)
+		return  err
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode == 200  {
+	body, err := ioutil.ReadAll(resp.Body)
+	} else {
+		return err
+	}
+
+	resp, e := http.Get(hipsterShopIstioManifestsURL)
+	if e != nil {
+		e = errors.Wrapf(e, "error parsing response body")
+		logrus.Error(e)
+		return  e
+	}
+	defer resp.Body.Close()
+	
+	if resp.StatusCode == 200  {
+		body, err := ioutil.ReadAll(resp.Body)
+		} else {
+			return err
+		}
+	
+	b, err := ioutil.ReadAll(resp.Body)
```

Comment:
One other request, the flow for both these URLs is pretty much the same. Can you please either put this in a loop? (OR) move the common code to a function and use that function in the two places to make the URL request and read the contents?